app/validators/users.js

```diff
@@ -10,17 +12,40 @@ const isPasswordLenghtvalid = password => {
 };
 
 const isPasswordAlphanumeric = password => {
-  if (regexAlphanumeric.test(password) === false) {
+  if (!regexAlphanumeric.test(password)) {
     return false;
   }
   return true;
 };
 
 const isEmailValid = email => {
-  if (regexValidEmail.test(email) === false) {
+  if (!regexValidEmail.test(email)) {
+    return false;
+  }
+  return true;
+};
+
+const isUserValid = user => {
+  const { email, password } = user;
+
+  if (!isPasswordLenghtvalid(password)) {
+    user.status = 'Password too short!';
```

Comment:
I get the point but this is not the best way. A better approach would be to return an error array instead of a boolean. 
So, if there are no errors you would return an empty array `[]`.
In the controller you would do something like

`if (!validate(user).errors.length) return console.log('There are no validation errors');`

So here in this code instead of returning each time a validation fails, you should push a propper message error to the errors array.